dedicated-examples/extended-error-tracking/index.js

```diff
@@ -0,0 +1,16 @@
+fixture('Log errors')
+    .page('./index.html')
+    
+test('Extended error tracking', async t => {
+   
+    await t
+            .click("#err1-btn")
+            .click("#err2-btn");
+
+    const errors = await t.eval(() => window.capturedErrors);
+    const foundCritical = errors.some(error => error.level === 'critical' && error.url === 'https://my-domain.example.com');
+    
+    await t
```

Comment:
Don't move to the new line, ff the only one action;
```js
await t..expect(foundCritical).notOk("Critical errors occurred during test"); 
```

Reply:
edited